examples/config-remote-session-store-redis.json

```diff
@@ -0,0 +1,35 @@
+{
+  "distSpecVersion": "1.1.1",
+  "storage": {
+    "rootDirectory": "/tmp/zot",
+    "remoteSessionCache": true,
+    "sessionDriver": {
```

Comment:
Shouldn't this be under UI as it is a UI specific setting?
@rchincha

Reply:
also be very descriptive in your commit msg about all the config and behavior changes happening in this PR

Reply:
Thanks! I've moved the config and placed it under UI config.
I've also updated the commit message with more details.